docs/DeveloperGuide.md

```diff
@@ -170,6 +170,33 @@ sequence diagram shows the interactions within `Model` when editing a person.
 
 ![EditSequenceDiagram-Model](images/EditSequenceDiagram-Model.png)
 
+### Delete Tag Command
+
+This section aims to show the logic behind delete tag command and the consideration behind the scene.
+
+Delete Tag Command is a new featured added for user to delete certain tag associated with a specific person located by index of the current list. This command is called by `deletetag` followed by the index of the targeted person, then a target tag `tag` object. In case of the targeting tag is not associated with the person, an error message suggesting missing targeted tag will be returned.
+
+The delete tag mechanism is facilitated by `DeleteTagCommand` and `DeleteTagCommandParser` that extends `Command` and `Parser` respectively. The `DeleteTagCommandParser` takes in an `index` and the `tag` to delete from a person. If both are supplied and valid, they are passed into the `DeleteTagCommand`.
+
+The current allowed formats for required fields are as follows:
+
+- `Index` : an `Integer` within the range greater than 0, smaller than the size of the currently displayed contact list.
+- `Tag` : one alphanumeric characters with no spaces between them.
+
+The following sequence diagram shows how `DeleteTagCommand` interacts with `Logic`.
+
+![DeleteTagSequenceDiagram](images/DeleteTagSequenceDiagram.png)
```

Comment:
Is it possible to save to SVG instead? We have decided to standardize to use .svg for our UML diagrams as they are more clear as compared to PNG.